src/styles/components/service-table/styles.less

```diff
@@ -37,6 +37,11 @@
         width: 115px;
       }
 
+      &-region {
+        max-width: 100%;
+        width: 140px;
```

Comment:
Would be nice if we could automatically handle column layout by relying on padding and document flow. Then we could avoid the need for hard-coding the width of each column in pixels every time we change the table.

This feedback isn't specific for this PR, but something to think about as we build out our [Table component](https://github.com/dcos-labs/ui-kit/tree/master/packages/table) and use it on more tables in DCOS.